examples/deadline/All-In-AWS-Infrastructure-Basic/ts/README.md

```diff
@@ -119,6 +119,30 @@ These instructions assume that your working directory is `examples/deadline/All-
     ```ts
     public readonly alarmEmailAddress?: string = 'username@yourdomain.com';
     ```
+
+15. Deadline Secrets Management is a feature used to encrypt certain values in the database that need to be kept secret. Additional documentation about the feature and how it works in the RFDK can be found in the [RFDK README](../../../../packages/aws-rfdk/lib/deadline/README.md). By default, Deadline Secrets Management is enabled, but it can be disabled by changing the `enableSecretManagement` variable in `package/config.ts`.
+
+    ```ts
+    public readonly enableSecretManagement?: boolean = false;
```

Comment:
```suggestion
    public readonly enableSecretsManagement: boolean = false;
```

Reply:
Thanks! It looks like we still need to remove the `?` from this line, since this variable isn't optional in the config file.